src/main/java/movies/Customer.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * @(#)Main.java
+ *
+ * Copyright (c) 2018 Jala Foundation.
+ * 2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+/**
+ * Customer of a video club that can rent movies get debt and renter points
+ *
+ * @version 1.0.
+ */
+package movies;
+
+import movies.movie.ChildrenMovie;
+import movies.movie.NewReleaseMovie;
+import movies.movie.RegularMovie;
+
+import java.util.Enumeration;
+import java.util.Vector;
+
+public class Customer {
+    private String name;
+    private Vector rented = new Vector();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the synchronized class "Vector" by an unsynchronized one such as "ArrayList" or "LinkedList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1149)